infinit/content.md

```diff
@@ -0,0 +1,67 @@
+Infinit’s storage platform transforms commodity servers into multiple flexible, scalable, secure and fault-tolerant storage logics tailored to your applications.
+
+## Usage
+
+The following commands show Docker-specific tips for a pleasant integration between Infinit and Docker. For the Infinit-specific documentation, consult [Infinit's website](https://infinit.sh/get-started).
+
+### Basics
+
+	docker run --rm infinit
+
+#### Sharing configuration folder
+
+Infinit stores its configuration under `/root/.local/share/infinit`. If you want to share this folder with your host, you can mount your host's configuration folder as a volume in your container.
+
+	docker run -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit
+
+*N.B. This might affect permissions of the mounted folder.*
+
+#### Using your user
+
+By default, Infinit uses the current user, `root` when running on a container. If you want to use your local user name, you can set the INFINIT_USER environment variable.
+
+	docker run -e INFINIT_USER=$USER -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit
+
+#### Accessing storage logics
+
+For now, the only logic available is a distributed file system.
+
+##### File system
+
+On UNIX-based systems, Infinit uses `FUSE` to provide mount points. Hence, for UNIX-based containers, you need `FUSE` installed on your host (see Infinit's [get-started page](https://infinit.sh/get-started#installation)) and run Docker in privileged mode (`--privileged`).
+
+	docker run --privileged -e INFINIT_USER=$USER -v ~/.local/share/infinit:/root/.local/share/infinit --rm infinit # volume mount --name <VOLUME> --mountpoint <DIRECTORY>
```

Comment:
For using FUSE inside a container, I think `--privileged` is a bit overkill (since it essentially allows _all_ privileges of the host, instead of the strictly-necessary subset).  Can we trim this down to just the subset necessary?  I'd imagine something like `--cap-add SYS_ADMIN --device /dev/fuse` is probably pretty close to the actual set required.